cypress/integration/pages/articles.js

```diff
@@ -120,12 +121,35 @@ const runTests = ({ service }) =>
       it('should render a formatted timestamp', () => {
         cy.request(`${config[service].pageTypes.articles.path}.json`).then(
           ({ body }) => {
-            if (body.metadata.language === 'en-gb') {
-              const { lastPublished } = body.metadata;
-              const timestamp = Cypress.moment(lastPublished).format(
-                'D MMMM YYYY',
-              );
-              cy.get('time').should('contain', timestamp);
+            const { language } = body.metadata.passport;
+            const { lastPublished } = body.metadata;
+            const { firstPublished } = body.metadata;
+            const updatedTimestamp = moment
+              .tz(lastPublished, `${appConfig[service].timezone}`)
+              .locale(language)
+              .format('D MMMM YYYY');
+            const firstTimestamp = moment
+              .tz(firstPublished, `${appConfig[service].timezone}`)
+              .locale(language)
+              .format('D MMMM YYYY');
+
+            if (service !== 'pashto') {
```

Comment:
Why do you have an exception here for Pashto? If it is because the Pashto work is being carried out in another issue, please add that as a comment here, so we remember to remove the exception.

Reply:
Please can you move this 'if pashto' logic to the top of the file with the others https://github.com/bbc/simorgh/pull/3131/files#diff-f4ac60f764604ddb9c71ebdb5e373708R14 please :) Easier to keep an eye on it